src/serverBuilder.ts

```diff
@@ -40,7 +40,7 @@ export class ServerBuilder {
   }
 
   private registerPreRoutesMiddleware(): void {
-    this.serverInstance.use(bodyParser.json());
+    this.serverInstance.use(bodyParser.json(this.config.get<bodyParser.Options>('server.request.payload')));
```

Comment:
we sure we want the entire object as parameter and not just the relevant value?

Reply:
all options parameters have default values, do you still think we should accept only the payload size?